main.css

```
@@ -0,0 +1,928 @@
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+  }
+  input[type=range]::-ms-fill-upper {
+    background:  rgba(255, 255, 255, 0.7);
+    border: 0px solid #000101;
+    border-radius: 50px;
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+  }
+  input[type=range]::-ms-thumb {
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    border: 0px solid #000000;
+    height: 20px;
+    width: 197px;
+    border-radius: 7px;
+    background: #ec9e03;
+    cursor: pointer;
+  }
+  input[type=range]:focus::-ms-fill-lower {
+    background:  rgba(255, 255, 255, 0.7);
+  }
+  input[type=range]:focus::-ms-fill-upper {
+    background:  rgba(255, 255, 255, 0.7);
+  }
+
+  @media (max-width: 642px) {
+    html {
+        max-width: 642px; 
+      }
+    body {
+        /* height: 1554px; */
+        width: 642px; 
+        margin: 0 auto;
+        /* background: #000000; */
+    }
+    .banner {
+        box-shadow: 0.105px 2.998px 5px rgba(0,0,0,0.75);
+        margin: 0;
+        background: rgba(245, 197, 6, 0.3);
+        line-height: 60px;
+        font-size: 19px;
+        letter-spacing: 0px;
+        color: #ffffff;
+        font-family: "Lucida Sans";
+        font-style: italic;
+        text-align: center;
+        text-shadow: 1px 1.732px 0px rgba(0,0,0,0.42);
+      }
+    .logo-banner {
+        margin: 0;
+        background-image: url("./assets/images/Logo\ \(mob\).svg");
+        width: 187px;
+        line-height: 60px;
+        background-position: 10px 9px;
+      }
+    .gradient {
+        background: linear-gradient(to bottom, #565656 0%, #383838 100%);
+    }
+    h1 {
+        line-height: 60px;
+        margin: 3px  0 -3px 60px;
+        
+        font-size: 19px;
+        letter-spacing: 0px;
+        color: #ffffff;
+        font-family: "Lucida Sans";
+        text-align: center;
+    }
+     
+    nav ul{
+        box-shadow: none;
+        margin: 0;  
+     }
+     nav {
+        display: flex;
+        margin: 0;  
+    }
+    
+    nav ul {
+        padding: 0;
+        display: flex;
+        list-style-type: none;
+        width: 456px;
+        height: 60px;
+        line-height: 60px;
+        background-image: linear-gradient(0deg, #f5c505 0%, #f5c505 100%);
+    }
+    
+    nav ul :first-child {
+        border: none;
+        margin-left: 0;
+    }
+    .nav-item:hover {
+        background: #ffffff;
+    }
+    .nav-item {
+        display: flex;
+        justify-content: center;
+        float: left;
+        border-left: 1px solid rgb(220, 176, 5);     
+        line-height: 41px;
+        /* margin-right: 30px; */
+        font-style: normal;
+        font-weight: bold;
+        font-size: 19px;
+        text-shadow: none;
+    }
+    nav ul li a {        
+        margin-top: 10px;
+        display: flex;
+        line-height: 41px;
+        text-decoration: none;
+    }
+    .aboutUsIcon {
+        background-image: url("./assets/images/Shape\ 5\ \(mob\).svg");
```

pretty strange file path
try to avoid spaces in file names
```suggestion
        background-image: url("assets/images/Shape5-mob.svg");
```